src/extension/background.js

```diff
@@ -50,10 +51,13 @@ const storeTheme = ({ theme }) => browser.storage.local.set({ theme });
 
 const applyTheme = ({ theme }) => {
   log("applyTheme", theme);
-  if (!theme) { return; }
+  if (!theme) {
+    return;
+  }
 
-  const backgroundImage = bgImages.keys().includes(theme.images.headerURL)
-    ? bgImages(theme.images.headerURL)
+  const background = theme.images.additional_backgrounds[0];
```

Comment:
Do we need to check if `theme.images.additional_backgrounds` isn't `undefined` or `null` here before trying to access the 0 index? (we seem to below in [./src/lib/metrics.js:116-118](https://github.com/mozilla/Themer/pull/141/files#diff-d0154faa85aed1ddcb0a1d33f0484caeR116))

Reply:
Should be okay - there's a call to [normalizeTheme](https://github.com/lmorchard/Themer/blob/311ede9fbfb3ed66fe4465ac25d1cd0148c3ef47/src/extension/background.js#L35) on the way in from the web, and the default is bg-000.svg if the set of dynamic imports don't contain the value